Lucid/Core/CoreManager.swift

```diff
@@ -935,7 +929,7 @@ private extension CoreManager {
 
         let initialUserAccess = context.userAccess
 
-        let result: AnySequence<E> = try await self.operationTaskQueue.enqueue { operationCompletion in
+        let result: AnySequence<E> = try await self.operationTaskQueue.enqueue {
```

Comment:
The ordering is still guaranteed since we process the operations one by one. We don't have a completion and we support executing in parallel but the execution order is given by the `queue` array. With this in mind it results impossible for an operation to "skip" the queue 

Now what's possible is that the second operation finishes before the first one, not sure If that's an issue. Otherwise we can change this to `barrier` and use the completion as we were using it prior to this change. 

There is not much of a difference anyway since we execute a couple of lines before calling completion on the previous version

Reply:
We should keep a close eye on this behavior, the two most important lines for ordering are:

```swift
let time = UpdateTime(timestamp: context.originTimestamp)
```
and

```swift
self.setUpdateTime(time, for: entitiesToUpdate.lazy.compactMap { $0.1?.identifier })
```

I'm not 100% certain on how `Task` blocks will handle this, and if this new code is safe. If we run into any issues, it's probably easiest just to make this a barrier and call the completions as we did before.

Reply:
Sounds like a plan!